src/shared/Map.ts

```diff
@@ -35,3 +35,20 @@ export type Landmark = z.TypeOf<typeof zLandmark>;
 export const zLandmarkScore = z.number().int().min(1).max(MAX_LANDMARK_SCORE);
 
 export type LandmarkScore = z.TypeOf<typeof zLandmarkScore>;
+
+export const zLandmarkScoreLog = z.object({
+    userId: z.string(),
+    landmark: z.string(),
+    createdAt: z.date(),
+    score: zLandmarkScore,
+    markdown: z.string().nullable(),
+});
+
+export type LandmarkScoreLog = z.TypeOf<typeof zLandmarkScoreLog>;
+
+export const landmarkScoreLogAttributes = [
```

Comment:
Move it to attributesAndIncludes.ts. This is not supposed to be shared with the frontend.